packages/plugin-react-native-global-error-handler/package.json

```diff
@@ -20,7 +20,7 @@
   "author": "Bugsnag",
   "license": "MIT",
   "devDependencies": {
-    "@bugsnag/core": "^7.0.0-pre-alpha-ben.6",
+    "@bugsnag/core": "^6.4.1",
```

Comment:
Should this be part of this PR?

Reply:
Yeah the explanation is in this commit: https://github.com/bugsnag/bugsnag-js/pull/646/commits/c342609cc603123ad636d4fa200929e93e6e81d5

It was needed to ensure the tests pass.